dist2src/cli.py

```diff
@@ -173,6 +212,19 @@ def run_prep(path):
                 logger.debug(str(line))
             raise
 
+        gitsetup = re.compile("^%gitsetup ", re.MULTILINE)
+        spec, number_of_subs = gitsetup.subn("%setup ", spec)
+        if number_of_subs > 1:
+            raise RuntimeError("Wow! Multiple %gitsetup macros in the spec file!")
+        if number_of_subs:
+            specfile_path.write_text(spec)
```

Comment:
Could `git checkout specfile_path` be used instead? I guess the intention is to undo all the changes done in the spec file in the lines above.

Reply:
+1, good one